mono/Makefile.am

```diff
@@ -16,6 +16,10 @@ if ENABLE_NETCORE
 btls_dirs = 
 endif
 
+if ENABLE_NETCORE
+SUBDIRS = eglib arch utils sgen metadata mini
+else
+
```

Comment:
Aren't the profilers needed ?

Reply:
that's unclear, we'll need to hook into existing profilers setup for netcore